FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -76,7 +76,7 @@ - (instancetype)initWithApp:(FIRApp *)app appInfo:(NSDictionary *)appInfo {
   self.authState = [FIRAppDistributionAuthPersistence retrieveAuthState:&authRetrievalError];
   // TODO (schnecle): replace NSLog statement with FIRLogger log statement
   if (authRetrievalError) {
-    NSLog(@"Found no tester auth token in keychain on intitialization");
+    NSLog(@"Found no tester auth token in keychain on intitialization. error? - %@", [authRetrievalError localizedDescription]);
```

Comment:
I don't think we want to error here - we expect this to find no tester in the keychain when they first start up. This isn't necessarily an error.

Reply:
Yeah. But I wanted to print the error just in case there was an actual error retrieving the keychain.